site/src/components/Calendar.vue

```diff
@@ -142,6 +144,18 @@ export default class Calendar extends Vue {
     }
   }
 
+  typeFullName(type: string): string {
+    const typeNames: { [key: string]: string } = {
+      LEC: "Lecture",
+      TES: "Exam",
+      REC: "Recitation",
+      LAB: "Lab",
+      STU: "Studio",
+    };
+
+    return typeNames[type] || "Unknown";
```

Comment:
I expect a lot of sections to not have a type, especially for older sections
Instead of unknown can we just not put anything